orienteer-core/pom.xml

```diff
@@ -106,6 +106,13 @@
 		    <artifactId>log4j</artifactId>
 	    </dependency>
 
+		<!-- ORIENTDB SQL GENERATOR DEPENDENCY-->
+		<dependency>
+			<groupId>com.github.raymanrt</groupId>
```

Comment:
If we are including this library into core - please make sure that it was deleted from orienteer-bpm/pom.xml